packages/list-store/src/useSearchParamsUpdate.ts

```diff
@@ -5,7 +5,7 @@ import {useEffect, useTransition} from 'react';
 import {useListStore} from './useListStore';
 import {getUrlWithSearchParams} from './search-params';
 
-export function useSearchParamsUpdate() {
+export function useSearchParamsUpdate({baseUrl = '/'} = {}) {
```

Comment:
I think I will delete the default (also in `getUrlWithSearchParams`). We don't use the default, and it gets passed through 3 functions. It will be clearer to make `baseUrl` not optional.